site/docs/types/compound_logical_types.md

```diff
@@ -7,13 +7,11 @@ Compound types include any type that is configurable including complex types as
 | FIXEDCHAR(L)                | A fixed length field of length L. L can be between [1..2,147,483,647]. Values less that are less in length than the length of the field are padded with spaces. | None                | None           | CharType(L)    | CHAR(L)                     |
 | VARCHAR(L)                  | A field that can holds UTF8 encoded strings between 0 and L length. The length of each value can be between [0..2,147,483,647]. The value of L can be between [1..2,147,483,647]. Values shorter than L are not padded. | None                | None           | VarcharType(L) | VARCHAR(L)                  |
 | FIXEDBINARY(L)              | A binary field that is fixed in width to L. Values that are shorter than L are 0-byte padded. | FixedSizeBinary&lt;L&gt;  | FIXED&lt;L&gt;       | -              | -                           |
-| DECIMAL(P,S)                | Fixed point decimal with precision P and scale S. Precision must be 38 or less. | DECIMAL(P,S)        | DECIMAL(P,S)   | DECIMAL(P,S)   | DECIMAL(P,S)                |
+| DECIMAL(P,S)                | A fixed precision decimal value having precision (P, number of digits) <= 38 and Scale (S, number of fractional digist) 0 <= S <=  P) | Decimal<P, S, bitwidth=128> | DECIMAL(P,S)   | DECIMAL(P,S)   | DECIMAL(P,S)                |
```

Comment:
```suggestion
| DECIMAL(P,S)                | A fixed precision decimal value having precision (P, number of digits) <= 38 and Scale (S, number of fractional digits) 0 <= S <=  P) | Decimal&lt;P, S, bitwidth=128&gt; | DECIMAL(P,S)   | DECIMAL(P,S)   | DECIMAL(P,S)                |
```
The json-markdown parser seems to allow <= but you could conceivably change that to &lt;= if you wanted to be thorough.